warmup/html.py

```diff
@@ -115,9 +115,11 @@ def html_histogram(data, nth):
   text-align: left;
 }
 table {
-  width: 100%%;
+  width: 1024px;
```

Comment:
Can I suggest we make this an `em` or `en` width, so that the table (slightly) adjusts to different font sizes?